Fix TopToon : update CSS selectors

MikeZeDev opened the pull request:
Fixes https://github.com/manga-download/hakuneko/issues/6905

MikeZeDev committed:
Fix TopToon : update CSS selectors

Fixes https://github.com/manga-download/hakuneko/issues/6905

MikeZeDev committed:
Update TopToon.mjs

27136773 commented on src/web/mjs/connectors/TopToon.mjs:
```diff
@@ -34,11 +34,11 @@ export default class TopToon extends Connector {
-            const subtitle = element.querySelector('p.episode_stitle');
-            title += subtitle ? ' - ' + subtitle.textContent.trim() : '';
+            let title = element.querySelector('p.ep_title').textContent.trim();
+            const subtitle = element.querySelector('p.ep_stitle');
```
Consider `element.querySelector('p.ep_stitle')?.textContent.trim()` to simplify the the following condition for ternary operator.

MikeZeDev replied:
I wont do that, because while nightly electron version support that syntax `element.querySelector('p.ep_stitle')?`, stable branch doesnt.  

The problem was first spotted on Tapytoon. Their login page use similar syntax, a user found people could login in nightly and not in stable branch because of that.

MikeZeDev committed:
Update TopToon.mjs

MikeZeDev committed:
attempt to fix lint error

MikeZeDev committed:
dont use optional chaining

optional chaining doesnt work in stable build because electron version is too old
